go/test/endtoend/vreplication/vreplication_test.go

```diff
@@ -973,12 +980,19 @@ func reshard(t *testing.T, ksName string, tableName string, workflow string, sou
 				t.Fatal(err)
 			}
 		}
+		tablets := vc.getVttabletsInKeyspace(t, defaultCell, ksName, "primary")
+
+		// Test multi-master setups with auto increment steps > 1.
+		for _, tablet := range tablets {
+			autoIncrementSetQuery := fmt.Sprintf("set auto_increment_increment = %d; set @@global.auto_increment_increment = %d",
```

Comment:
Nit, but IMO we should be explicit in both cases about the intended scope. i.e.:
```
autoIncrementSetQuery := fmt.Sprintf("set @@session.auto_increment_increment = %d; set @@global.auto_increment_increment = %d",
```